src/pages/instances/Events.tsx

```diff
@@ -47,6 +49,7 @@ const Events: FC = () => {
         void queryClient.invalidateQueries({
           queryKey: [queryKeys.operations, event.project],
         });
+        void refetchOperations();
       }
       setTimeout(() => handleEvent(event), 1000); // handle with delay to allow operations to vanish
```

Comment:
I think this timeout can be removed now. we can directly call handleEvent without a delay.

Reply:
what was the reason that this was needed in the first place?

Reply:
I think it was added because of some timing issues on the instance list for instances on creation. But that code is much more robust now and checking operation status as well as if the instance is in the get instances api response to avoid duplicates. Hence I suggest to make this instant without the timeout.